### DISCUSSION TITLE
support CDN for container queries

### DISCUSSION THREAD

**Discussion by HananoshikaYomaru** [tailwindlabs/tailwindcss#12074] [Ideas]
https://[cdn.tailwindcss.com/?plugins=container-queries](https://cdn.tailwindcss.com/?plugins=container-queries)

container queries is first party plugin but cannot install through CDN

<img width="778" alt="CleanShot 2023-09-24 at 00 45 51@2x" src="https://github.com/tailwindlabs/tailwindcss/assets/43137033/c33a7734-1bdb-4c70-a84d-e0b2559cab25">

**Comment by jdocksey** [comment]
I'd like to be able to use the `container-queries` plugin through the CDN as well. It would be a major help with prototyping designs.

**Comment by elia** [comment]
We were able to add them using import maps:

```html
<script type="module">
  import containerQueries from '@tailwindcss/container-queries'
  const {defaultTheme} = tailwind
  tailwind.config = {
    theme: {…},
    plugins: [
      containerQueries(),
    ],
  }
</script>
```

I wonder tho if there's any chance to have the source of the PlayCDN available, we've found that is very useful for tinkering with classes live in the browser during development.
